less/availity-cards.less

```diff
@@ -63,6 +91,29 @@
   }
 }
 
+.card-header-link {
```

Comment:
Sure. That works
On Sat, Sep 12, 2015 at 4:01 PM robert mcguinness <notifications@github.com>
wrote:

> In less/availity-cards.less
> <https://github.com/Availity/availity-uikit/pull/100#discussion_r39340863>
> :
>
> > @@ -63,6 +91,29 @@
> >    }
> >  }
> >
> > +.card-header-link {
>
> I added .card-header-link and .card-title-link which basically do the
> same thing. would that be sufficient?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Availity/availity-uikit/pull/100/files#r39340863>.
>